build.gradle

```diff
@@ -155,6 +155,12 @@ ext.encoding = {
 ext.schedulingFunctionalTestConfiguration = {
 
     include 'functionaltests/**'
+    
+    boolean isRegression = System.getProperty("regression") != null ? Boolean.valueOf(System.getProperty("regression")) : false;
```

Comment:
Why not using Gradle project properties which seems done for this purpose (instead of `System.getProperty(...)`)? 

```gradle
project.hasProperty('regression')
```

then invoke with:

`-Pregression` ?